package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "lfg-omega13",
-  "version": "0.1.1",
-  "description": "A big red button that sends a very aggressive phone notification.",
+  "version": "0.1.2",
```

Comment:
Actually, now that I think about it, don't change the package version here. I'll change it when I do the release. Amend the commit to leave it as `0.1.1` for now.